src/libfetchers/cache.cc

```diff
@@ -36,7 +36,7 @@ struct CacheImpl : Cache
     {
         auto state(_state.lock());
 
-        auto dbPath = getCacheDir() + "/nix/fetcher-cache-v2.sqlite";
+        auto dbPath = getCacheDir() + "/fetcher-cache-v3.sqlite";
```

Comment:
```suggestion
        auto dbPath = getCacheDir() + "/nix/fetcher-cache-v3.sqlite";
```
since 2.24 doesn't have 38bfbb297c380f8b07d8a20ffdeb72da71c1567c.